### PR TITLE
Enable tide reviewApprovedRequired on knative-sandbox/net-contour

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: prow/knative/config.yaml [hunk 1]
```diff
@@ -173,6 +174,18 @@ tide:
     missingLabels: *knative_tide_missing_labels
     author: knative-prow-robot
     reviewApprovedRequired: false
+  - repos:
+    - knative-sandbox/net-contour
+    labels:
+    - lgtm
+    - approved
```
Comment (n3wscott) [inline prow/knative/config.yaml:181]: will github reviews make the approved label? I don't think so, but then will prow add approved label from reviews? I thought that is what @grantr added.

Comment (chaodaiG) [reply prow/knative/config.yaml:181]: I think prow will add approved labels, but your change seems to make this repo requires `lgtm` and `approve` label, as well as explicit github code review approval, is this desired?

**Comment by chaodaiG** [comment]
/approve

/hold
If you want to address my nits.

File: prow/knative/config.yaml (hunk 1 above)
Comment (n3wscott) [reply prow/knative/config.yaml:181]: yes, that is the goal

File: prow/knative/config.yaml [hunk 2]
```diff
@@ -173,6 +174,18 @@ tide:
...
+    - "cla: yes"
+    missingLabels: &knative_tide_missing_labels
```
[the remark this answers is not in the document]
Comment (n3wscott) [reply prow/knative/config.yaml:183]: oh yeah don't want do to that, good catch.

**Pull request merged**